docs/components/networktransform.md

```diff
@@ -50,28 +45,90 @@ With such nested NetworkTransforms you can theoretically have a NetworkTransform
 :::
 
+### When to nest NetworkTransforms
+
+If you plan on having unique motion relative to a parent NetworkTransform should you consider nesting NetworkTransforms. As an example, if you are using a NetworkAnimator component to synchronize animations then you need not worry about synchronizing each child node of the model animating as the animation system and NetworkAnimator are handling this for you. 
+
+Really, if you can get away with allowing the parent to synchronize its transform then all child transforms will move, rotate, and scale with the parent automatically. As an example, if you wanted some object to orbit another object and used the server time (i.e. NetworkManager.ServerTime) to feed into something like a sine or cosine based calculation and used that on all instances (local and remote) then you would have no reason to synchronize the motion of the child orbiting the parent since you are feeding the algorithm a value already being synchronized with all clients (i.e. the server time).
+
+:::tip
+**The general rule to follow is:**
+
+Unless you have some unique motion that needs to be applied to a child object and it can't be synchronized through an animation or an algorithm based on the synchronized time, then you should nest NetworkTransforms.
+
+However, nesting NetworkTransforms is a more optimal path then creating a bunch of individual single node network prefabs with NetworkTransforms and parenting those once they are spawned.
+:::
 
 ## Configuring
 
 When you select a NetworkTransform component, you will see the following properties in the inspector view:
 
-![image](/img/NetworkTransformProperties.png)
+![image](/img/networktransform/NetworkTransformProperties.png)
 
 ### Property synchronization
 
 Some of the `NetworkTransform` properties are automatically synchronized by the authoritative instance to all non-authoritative instances.  It is **important to note** that when any synchronized property changes the NetworkTransform is effectively "teleported" (i.e. all values re-synchronized and interpolation is reset) which can cause a single frame delta in position, rotation, and scale (depending upon what is being synchronized). _Always keep this in mind when making adjustments to NetworkTransform properties during runtime._
 
-### Synchronizing
+#### Properties that can cause a full state update (i.e. teleport)
+The following are a list of `NetworkTransform` properties that will cause a full state update (*i.e. effectively a teleport*) when changed during runtime by the authority instance: 
+- UseUnreliableDeltas
+- Interpolate
+- Slerp Position
+- Use Quaternion Synchronization
+- Use Quaternion Compression
```

Comment:
I can't see this property on the new screenshot you've provided. Intentional?

Reply:
I updated the list and linked each field to its corresponding API documentation.
Do you think that should help with the context?